main.cpp

```diff
@@ -130,16 +134,6 @@ struct Edge{
 
 void solve(){
     
-    
-    list<int> lst;
-    lst.pop_back();
-    list<int>::iterator it = lst.begin();
-    reverse(begin(lst),end(lst));
```

Comment:
this is a test

>  list<int> lst;
    lst.pop_back();
    list<int>::iterator it = l

aa 
aa

Reply:
ok roger that

Reply:
yeah thats true, but we can make it better by understanding that

Reply:
ok

Reply:
aaaaa